File: rockPaperScissors.js
var userInput = 'Rock';
userInput = userInput.toLowerCase();
var userInput = 'Paper';
userInput = userInput.toLowerCase();
var userInput = 'Scissors';
userInput = userInput.toLowerCase();

function getComputerChoice() {
  var randomNumber = Math.floor(Math.random() * 3);
  switch(randomNumber) {
    case 0:
      return 'rock';
    case 1:
      return 'paper';
    case 2:
      return 'scissors';
  }
}

function determineWinner(userChoice, computerChoice) {
  if  (userChoice === computerChoice) {
    return 'it\'s a tie!'
  } else if (userChoice === 'rock') {
    if(computerChoice === 'paper') {
      return 'computer wins!';
    } else {
      return 'you win!';
    }
  }
  if (userChoice === 'paper') {
    if (computerChoice === 'scissors') {
      return 'computer wins!';
    } else {
      return 'you win!';
    }
  }
  if (userChoice === 'scissors') {
    if (computerChoice === 'paper') {
      return 'you win!';
    } else {
      return 'computer wins!';
    }
  }
}

function playGame() {
  var userChoice = userInput;
  var computerChoice = getComputerChoice();
  console.log('You threw: ' + userChoice);
  console.log('The computer threw: ' + computerChoice);
  console.log(determineWinner(userChoice, computerChoice));
}
playGame();

File: rock_paper_scissors.cpp
#include <iostream>
#include <stdlib.h>
//Rock Paper Scissors Program

int main(){
  //Live long and prosper
  srand(time(NULL));
  //computer picks a random number between 1 and 3
  int computer = rand() % 3 + 1;
  //user input
  int user = 0;
  //user prompts
  std::cout << "====================\n";
  std::cout << "rock paper scissors!\n";
  std::cout << "====================\n";
  std::cout << "1) ✊\n";
  std::cout << "2) ✋\n";
	std::cout << "3) ✌️\n";
  std::cout << "Shoot!\n";
  std::cin >> user;
  
  //Determine the winner
  //Prints what the computer chose
  std::cout << "Computer picked " << computer << "\n";
  if(user == computer){
    std::cout << "It's a tie!\n";
  }
  else if(user == 1){
    if(computer == 2){
      std::cout << "Paper beats rock! Computer wins!\n";
    } else{
      std::cout << "Rock beats scissors! You win!\n";
    }
  }
  else if(user == 2){
    if(computer == 1){
      std::cout << "Paper beats rock! You win!\n";
    } else{
      std::cout << "Scissors beats papaer! Computer wins!\n";
    }
  }
  else if(user == 3){
    if(computer == 2){
      std::cout << "Scissors beats paper! You win!\n";
    } else{
      std::cout << "Rock beats scissors! Computer wins!\n";
    }
  }
}
